packages/credentials/verify/src/verification/nftStatus.ts

```diff
@@ -0,0 +1,42 @@
+import { VcNft } from "../types/verifiableCredential.js";
+import { Contract } from "ethers";
+import { abi_ERC_721 } from "../ABI/ERC721.js";
+import { constants } from "ethers";
+
+import {
+    JsonRpcProvider,
+    StaticJsonRpcProvider,
+} from "@ethersproject/providers";
+
+// const POLYGON_RPC_URL = "https://rpc-mainnet.maticvigil.com/";
+const POLYGON_RPC_URL = "https://rpc-mumbai.maticvigil.com/";
```

Comment:
this stuff should be configuarable, see the `environment` param we use in other packages